doc/basics/dependencies.md

```diff
@@ -85,21 +85,21 @@ For convenience, dependencies install procedure create a configuration file, you
 The cmake config file `/path/to/external/install/radium-options.cmake` contains, depending on your dependency configuration:
 
 ~~~{.cmake}
-set(RADIUM_DEP_PREFIX "/path/to/external/install/")
-set(Eigen3_DIR        "${RADIUM_DEP_PREFIX}/share/eigen3/cmake/" CACHE PATH "My Eigen")
-set(OpenMesh_DIR      "${RADIUM_DEP_PREFIX}/share/OpenMesh/cmake/" CACHE PATH "My OpenMesh")
-set(cpplocate_DIR     "${RADIUM_DEP_PREFIX}/share/cpplocate/" CACHE PATH "My cpplocate")
-set(glm_DIR           "${RADIUM_DEP_PREFIX}/lib/cmake/glm/" CACHE PATH "My glm")
-set(glbinding_DIR     "${RADIUM_DEP_PREFIX}/share/glbinding/" CACHE PATH "My glbinding")
-set(globjects_DIR     "${RADIUM_DEP_PREFIX}/share/globjects/" CACHE PATH "My globjects")
-set(stb_DIR           "${RADIUM_DEP_PREFIX}/include/stb/" CACHE PATH "My stb")
-set(assimp_DIR        "${RADIUM_DEP_PREFIX}/lib/cmake/assimp-5.0/" CACHE PATH "My assimp")
-set(tinyply_DIR       "${RADIUM_DEP_PREFIX}/lib/cmake/tinyply/" CACHE PATH "My tinyply")
-set(tinyEXR_DIR       "${RADIUM_DEP_PREFIX}/share/tinyEXR/cmake/"  CACHE PATH "My tinyEXR")
-set(nlohmann_json_DIR "${RADIUM_DEP_PREFIX}/lib/cmake/nlohmann_json/" CACHE PATH "My json")
-set(PowerSlider_DIR   "${RADIUM_DEP_PREFIX}/lib/cmake/PowerSlider/" CACHE PATH "My PowerSlider")
-set(RADIUM_IO_ASSIMP   ON CACHE BOOL "Radium uses assimp io")
-set(RADIUM_IO_TINYPLY  ON CACHE BOOL "Radium uses tinyply io")
+set(Eigen3_DIR "/path/to/external/install/share/eigen3/cmake/" CACHE PATH "My Eigen3 location")
+set(OpenMesh_DIR "/path/to/external/install/share/OpenMesh/cmake/" CACHE PATH "My OpenMesh location")
+set(cpplocate_DIR "/path/to/external/install/share/cpplocate/" CACHE PATH "My cpplocate location")
+set(nlohmann_json_DIR "/path/to/external/install/lib/cmake/nlohmann_json/" CACHE PATH "My nlohmann_json location")
+set(glm_DIR "/path/to/external/install/glm/lib/cmake/glm/" CACHE PATH "My glm location")
+set(glbinding_DIR "/path/to/external/install/glbinding/" CACHE PATH "My glbinding location")
+set(globjects_DIR "/path/to/external/install/globjects/" CACHE PATH "My globjects location")
+set(stb_DIR "/path/to/external/install/include/stb/" CACHE PATH "My stb location")
+set(stb_INCLUDE_DIR "/path/to/external/install/include/" CACHE PATH "My stb_INCLUDE location")
+set(tinyEXR_DIR "/path/to/external/install/share/tinyEXR/cmake/" CACHE PATH "My tinyEXR location")
+set(assimp_DIR "/path/to/external/install/lib/cmake/assimp-5.0/" CACHE PATH "My assimp location")
+set(tinyply_DIR "/path/to/external/install/lib/cmake/tinyply/" CACHE PATH "My tinyply location")
+set(PowerSlider_DIR "/path/to/external/install/lib/cmake/PowerSlider/" CACHE PATH "My PowerSlider location")
+set(RADIUM_IO_ASSIMP ON CACHE BOOL "Radium uses assimp io")
+set(RADIUM_IO_TINYPLY ON CACHE BOOL "Radium uses tinyply io")
```

Comment:
Why removing the definition of the common prefix set(RADIUM_DEP_PREFIX "/path/to/external/install/") and its usage in the definition of all the locations. ?
The generated radium-options.cmake should also do this no ?

Reply:
I mean it's easier to store directly dep_DIR for further dependencies configuration and store a list of dep to generate radium-options.cmake, instead of creating dep_SUFFIX...
And since this file is generated and not meant to be edited by hand, I think it's not really important to make it more readable.
We can also remove its content from the doc.

Reply:
As I said, this PR seems fine for me and I agree to merge. But perhaps we can go further in improving documentation, configuration and installation of Radium-engine related stuff. 

Well, the file can also be generated differently with the same objectives in a more human readable  way, which can be helpful in some circumstances, without more effort than a tiny adaptation of your proposal. 
The generation of the file is a good thing, but relying on cmake `configure_file` (https://cmake.org/cmake/help/latest/command/configure_file.html) with variables substitution will help to simplify and generate something more robust : 
  - `set_external_dir` do not prepend the external installation path with `${CMAKE_INSTALL_PREFIX}` (line 48 of cmake/ExternalInclude.cmake) so that externals defines their path in a relative way according to an unknown prefix.
  - lines 105 to 114 of `externals/CMakeLists.txt` the script generate a file `radium-options.cmake.in` with the first `(set RADIUM_DEP_PREFIX \"@CMAKE_INSTALL_PREFIX@\"`  and generates the lines for the configured externals by appending `"set(${external}_DIR \"${RADIUM_DEP_PREFIX}/${${external}_DIR}\" CACHE PATH \"My ${external} location\")\n"`  
  - once the `radium-options.cmake.in` is filled, the script call `configure_file(radium-options.cmake.in radium-options.cmake @ONLY)`
  - the file `radium-options.cmake` is then installed using `install(FILES radium-options.cmake TYPE DATA)` 
  - the installed file will then be available in the directory `/path/to/external/install/share`

Relying on "install" to put the `radium-options.cmake` to the right place will need to install externals explicitly and not during build using the sequence `cmake build ..... && cmake install ...` as for the Radium libs themselves.
If you think externals should be installed automatically at build -time, the `radium-options.cmake` could be copied at the right place using `file(COPY .....)` instead of generated directly at this location but this is not the standard way to do that. 

But installing explicitly is more secure to relative paths versus absolute path handling and can be related to the recommandation of the documentation at https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#generating-a-package-configuration-file saying  ``` ...  under Windows and OSX, where users are used to choose the install location of a binary package at install time, independent from how `CMAKE_INSTALL_PREFIX` was set at build/cmake time. ``` 
You fix a problem related to this in this PR (by using `TYPE` instead of `DESTINATION` to install doc and licence files, so, why not following the same principle for externals ?

I'll be able to push an implementation of this proposal in the next few days and will be available to discuss that AFK if you prefer.

Reply:
I have referenced your concern in a new issue, #1017 , please accept the PR as you are agree to merge.
Other improvements or proposals are welcome in upcoming PR.